service/history/execution/mutable_state_builder.go

```diff
@@ -4690,8 +4628,7 @@ func (e *mutableStateBuilder) closeTransactionHandleActivityUserTimerTasks(
 	transactionPolicy TransactionPolicy,
```

Comment:
It seems that we can remove this parameter

Reply:
Thanks, updated.